docs/explanation/features/remote-script-execution.md

```diff
@@ -0,0 +1,73 @@
+(explanation-remote-script-execution)=
+
+# Landscape remote script execution
+
+This document explains how Landscape Client executes scripts.
+
+## Overview
+
+When an administrator requests a script execution, or when a script profile is scheduled, Landscape Server creates an `ExecuteScriptRequest` activity. See {ref}`explanation-activities` for details on how activities are delivered to clients.
+
+## Attachments
+
+Scripts may include attachments. Attachments are stored on Landscape Server, and clients can fetch them before execution.
+
+## `execute-script` message
+
+Landscape Server sends a message to Landscape Client in the following form:
+
+```json
+{
+  "type": "execute-script",
+  "interpreter": "INTERPRETER",
+  "code": "SCRIPT_CODE",
+  "username": "USER",
+  "time-limit": "TIME_LIMIT",
+  "operation-id": "ACTIVITY_ID",
+  "attachments": ["ATTACHMENT_IDS"],
+  "env": {
+    "LANDSCAPE_ACCOUNT": "ACCOUNT_NAME",
+    "LANDSCAPE_COMPUTER_ID": "COMPUTER_ID",
+    "LANDSCAPE_COMPUTER_TAGS": ["COMPUTER_TAGS"],
+    "LANDSCAPE_URL": "LANDSCAPE_URL",
+    "LANDSCAPE_ACTIVITY_ID": "ACTIVITY_ID",
+    "LANDSCAPE_ACTIVITY_CREATOR_ID": "CREATOR_ID",
+    "LANDSCAPE_ACTIVITY_CREATION_TIME": "CREATION_TIME"
+  }
+}
+```
+
+Field descriptions:
+
+`interpreter`: The interpreter to run the script. This is parsed from the script code.
+`code`: The script body, excluding the interpreter line.
+`username`: The user under which the script will run. In the Landscape Client snap, this is always root.
+`time_limit`: Maximum execution time before the process is forcibly terminated.
+`activity_id`: Unique identifier of the activity.
+`attachments`: List of attachment IDs stored on Landscape Server.
+`env`: Environment variables provided by Landscape Server, including account name, computer metadata, and activity metadata.
```

Comment:
```suggestion
- `interpreter`: The interpreter to run the script. This is parsed from the script code.
- `code`: The script body, excluding the interpreter line.
- `username`: The user under which the script will run. In the Landscape Client snap, this is always root.
- `time_limit`: Maximum execution time before the process is forcibly terminated.
- `activity_id`: Unique identifier of the activity.
- `attachments`: List of attachment IDs stored on Landscape Server.
- `env`: Environment variables provided by Landscape Server, including account name, computer metadata, and activity metadata.
```
Otherwise it gets formatted like a paragraph